invenio_records_resources/services/records/queryparser/transformer.py

```diff
@@ -33,15 +33,55 @@ def term_name(self):
         """Get the term name."""
         return self._term_name
 
-    def map_word(self, node):
+    def map_word(self, node, **kwargs):
```

Comment:
for backwards compat

Reply:
That is not true no? Basically we change the signature and if someone has implemented their class and configured it on their instance then it will give an error...